xradar/model.py

```diff
@@ -241,6 +241,8 @@
         ("pulse_width", None),
         ("antenna_gain_h", None),
         ("antenna_gain_v", None),
+        ("ant_gain_h", None),  # gamic
```

Comment:
We will override the non-conforming name with the standard name like this:

```suggestion
        ("ant_gain_h", "antenna_gain_h"),  # gamic
```

Reply:
Same adaptions needed for the rest of the newly added namings. Not sure what to do if there are is no equivalent in the standard model.

Reply:
@aladinor Can you do this kind of fixing for the other new entries, so that they get converted/renamed to the standard naming?